android-components/samples/browser/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,10 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-feature
```

Comment:
Leaving a comment here, but it applies to all the other apps in this commit:

So I think this is not enough for us.

We use the camera permission for the front/back/external cameras.

What I'm reading on [the docs here](https://developer.android.com/guide/topics/manifest/uses-feature-element#hw-features), is that this permission is only for the back camera.

We either need to use `android.hardware.camera.any` instead, or we need to add three separate `uses-feature` for `android.hardware.camera`, `android.hardware.camera.front`, and `android.hardware.camera.external`. It's not clear to me from this doc at least which one we need.

Reply:
This was added because the manifest specifies a permission that [implies](https://developer.android.com/topic/arc/manifest#implied-features) a feature requirment. Meaning that without specifying that the feature is not required, it would imply that it is required, and the app would not be able to be found on devices without that specific hardware (in our case Chromebook).
We could also suppress the lint warning but this seems more clean.

Reply:
> Meaning that without specifying that the feature is not required, it would imply that it is required, and the app would not be able to be found on devices without that specific hardware (in our case Chromebook).

Even then, we would need to say that we need (non-required) hardware to all cameras in web content, right?

Reply:
Added a comment and commit message to make this more clear for future references.